about.html

```diff
@@ -12,57 +12,69 @@
 </head>
 
 <body>
-    <div class="container about-page">
```

Comment:
In my opinion, this is the cleanest way to do the header/nav area.  By putting the image inside the `nav` element we can control it just like we control the anchors.

Reply:
Nice I didnt realize I did it like that.